setup.py

```diff
@@ -25,6 +27,17 @@
 if py_version >= (3, 5):
     packages.append('websockets/py35')
 
+ext_modules = []
+if (platform.python_implementation() == 'CPython' and
+        os.environ.get('WEBSOCKETS_CEXT_BUILD') != '0'):
```

Comment:
Done. Tests are now cleaner (ensure coverage with extension built) as in your PR.